test/Mono.Linker.Tests.Cases/FeatureSettings/Dependencies/ResourceFile.txt

```diff
@@ -0,0 +1 @@
+This resource will be embedded into an assembly and then removed by the linker.
```

Comment:
Is this file actually used anywhere (I probably missed it)

Reply:
Yeah - it's used in https://github.com/mono/linker/pull/1277/files#diff-7158670bf7e2f9050db3d3309f790831R7 since I moved the feature switch tests to a new directory. The substitutions tests have their own copy which is why this isn't just a rename like the others.